.github/workflows/error-ref-publisher.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
           commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
           commit_options: '--signoff'
           commit_message: 'run error codes utility'
-          file_pattern: helpers/ **.go
+          file_pattern: helpers/ **/error.go
```

Comment:
@iamsdas Are you sure this would include `error.go` files from all directories?

Reply:
I tested it out and it was working for me so it should work🤞

Reply:
Ok cool. Let's get this in then : )